mods/sp/rules/infantry.yaml

```diff
@@ -1163,6 +1181,31 @@ COLOSSI:
 	RejectsOrders@deployment:
 		Reject: AttackMove, AssaultMove
 		RequiresCondition: deployed
+ 
+COLOSSI_CHRONO:
```

Comment:
This is missing the `Selectable:Class` override to disguise it as a standard unit when controlled by a player.

Reply:
Thanks. and this unit need a conditional arm for current weapon and old weapon due to Selectable can used as disguise.